File: JUNGOL/Beginner/#5932 숫자사각형4-2/숫자사각형4-2.py
n = int(input())

dir = 1
for i in range(n):
    if dir == 1:
        for j in range(1, n+1):
            print(j, end=' ')
        print()
    else:
        for j in range(n, -1, -1):
            print(j, end=' ')
        print()
    dir = -dir

File: JUNGOL/Beginner/#5932 숫자사각형4-2/숫자사각형4-2.cpp
#include <iostream>
using namespace std;

int main(){
    int n; cin>>n;

    int dir = 1;
    for(int i=0; i<n; i++){
        if(dir == 1){
            for(int i=1; i<=n; printf("%d ", i++));
            cout<<endl;
        }
        else{
            for(int i=n; i>0; printf("%d ", i--));
            cout<<endl;
        }
        dir = -dir;
    }
}
